database/migrations/2020_12_29_025357_create_modules_table.php

```diff
@@ -27,6 +27,7 @@ public function up()
             $table->text('aliases')->nullable();
             $table->text('files')->nullable();
             $table->text('requires')->nullable();
+            $table->integer('priority')->default(0);
```

Comment:
Where should this come into play?

Reply:
In some module.json files contain "priority" attribute, I decided to create database structure supports it first. I'm fully aware I need nWidart and other experienced contributors to review and support me this PR.

Reply:
Alrighty

Reply:
If you find out what should be improved or need more work to do on this feature please let me know.  I would appreciate your help.

Reply:
I'll take a look at some point today or tomorrow, and trying it out, going to try it with multitenancy, which is one of the good use cases this could be use for.

Reply:
Yes my project is multi-tenancy. And I was a little bit surprised laravel-modules only supports local file management.

Reply:
Just out of curiosity is your project a single db multi tenant?

Reply:
It was not decided yet by my company.